tests/fixtures/pagination_compat/index.liquid

```diff
@@ -0,0 +1,25 @@
+---
+permalink: /
+layout: default.liquid
+pagination:
+  include: All
```

Comment:
What is the goal of this test?

Reply:
To make sure that we can use `collections.posts`. Before this PR, if we have pagination enabled, `collections` was not created.

Reply:
Unless legacy support is on, there shouldn't be a `collections` though

Reply:
Yes, but you said it was activated by default. 
This test ensure that if the legacy support is disable, we don't have `collections`: https://github.com/cobalt-org/cobalt.rs/blob/5d3a3bfb882325fef9f084a4041760892a60acfe/tests/mod.rs#L328

Reply:
Ok, so this is just to ensure `collections.posts` didn't break?  Don't we already have tests for that?

Reply:
>  Before this PR, if we have pagination enabled, collections was not created.

That's why I have this test.
It's for the case "when pagination is enabled, we still have access to collections.posts" .
It wasn't the case before this whole transition discussion: if pagination was enabled, collections.posts was not created for pages.